src/samples/Sonar.hpp

```diff
@@ -114,7 +114,11 @@ struct Sonar
     /** The speed of sound in water at the time of acquisition and in m/s*/
     float speed_of_sound;
     
-    /** Number of bins in a beam */
+    /**
+     * Number of bins in a beam
+     * 
+     * @deprecated, use getBeamBinCount() to return the number of bins in each beam
```

Comment:
`@deprecated this field seem to be filled inconsistently across packages, and actually does not provide useful information. Use \c getSingleBeamBinCount() instead`